.github/workflows/ci.yml

```diff
@@ -81,7 +85,8 @@ jobs:
               run: |
                   python -m pip install --upgrade pip
                   python -m pip install poetry
-                  python -m poetry install --extras='cdk datasets-endpoint'
+                  python -m poetry install \
+                      --extras='cdk datasets-endpoint check-metadata-content-job'
```

Comment:
I found one thing - name of Poetry extra is not matching directory name of batch job
11:49 PM
check-metadata-content-job should be check-stac-metadata-job